repos/system_upgrade/el8toel9/actors/sssdfacts/libraries/sssdfacts8to9.py

```diff
@@ -0,0 +1,53 @@
+from leapp.models import SSSDConfig8to9
+
+
+class SSSDFactsLibrary(object):
+    """
+    Helper library from SSSDFacts actor to allow unit testing.
+    """
+    def __init__(self, config):
+        self.config = config
+
+    def get_fallback(self, section, key, fallback):
```

Comment:
Maybe instead of having 2 dedicated functions just use configparser's fallback option inplace?
Like change occurrences  of get_fallback to `self.config.get(section, key, fallback=fallback)` and getboolean_fallback change to `self.config.getboolean(section, key, fallback=fallback)`?

https://docs.python.org/3/library/configparser.html#fallback-values